libraries/logging/iot_logging_task_dynamic_buffers.c

```diff
@@ -153,10 +153,17 @@ void vLoggingPrintf( const char * pcFormat,
 
     if( pcPrintString != NULL )
     {
+        /* Variable to track when a stream of log message is completed. */
+        /* Note: A log message may be sent over multiple calls to vLoggingPrintf, thus, */
+        /* this variable tracks when the log message ends. */
+        static BaseType_t xEndOfMessage = pdTRUE;
```

Comment:
How does this static variable handle multiple tasks calling vLoggingPrintf()?

Reply:
Also, is `UBaseType_t` a more appropriate type for this variable?

Reply:
Can this be deleted now?

Reply:
Deleted

Reply:
Deleted